server/endpoint/health/searcher/endpoint_test.go

```diff
@@ -367,7 +367,7 @@ func Test_Health_Endpoint(t *testing.T) {
 			p := filepath.Join("testdata", normalizeFileName(tc.name)+".golden")
 
 			if *update {
-				json, err := json.Marshal(endpointResponseTyped)
+				json, err := json.MarshalIndent(endpointResponseTyped, "", "    ")
```

Comment:
Using `MarshalIndent` here so the golden file is easier to read and `diff`-able.